src/app/lib/views/title_bar.js

```diff
@@ -71,12 +71,7 @@
         },
 
         minimize: function () {
-            var that = this.nativeWindow;
-            if (AdvSettings.get('minimizeToTray')) {
-                minimizeToTray();
-            } else {
-                that.minimize();
-            }
+            this.nativeWindow.minimize();
```

Comment:
so that fires the 'minimize' event we handle later ? it looks wierd and i'm not sure i can test that on my setup (i have no tray)